features/step_definitions/tournament.rb

```diff
@@ -96,3 +96,14 @@ def find_team_of_match(match_gameday, match_num, home_or_away)
 Given(/^a tournament with gamemode (.*)$/) do |mode|
```

Comment:
While a little bit off for this pr, you should actually use the except syntax [there](https://github.com/hpi-swt2/sport-portal/pull/274/files#diff-1b3862269b88b8c332be42cbd071b46eR79).